ffi-build.sh

```diff
@@ -23,15 +23,15 @@ target="$(rustc -vV | awk '/^host:/ { print $2 }')"
 # provided. At least on Alpine, libgcc_s may not even exist in the users'
 # images, so -static-libgcc is recommended there.
 case "$target" in
-    "x86_64-alpine-linux-musl")
+    "x86_64-alpine-linux-musl"|"aarch64-alpine-linux-musl")
```

Comment:
I agree that Linux is sufficiently general in terms of the x86 and ARM builds that handling both in the same section is appropriate.